stac_mjx/io_dict_to_hdf5.py

```
@@ -0,0 +1,67 @@
+def load(filename, ASLIST=False):
+    """
+    Default: load a hdf5 file (saved with io_dict_to_hdf5.save function above) as a hierarchical
+    python dictionary (as described in the doc_string of io_dict_to_hdf5.save).
+    if ASLIST is True: then it loads as a list (on in the first layer) and gives error if key's are not convertible
+    to integers. Unlike io_dict_to_hdf5.save, a mixed dictionary/list hierarchical version is not implemented currently
+    for .load
+    """
+    with h5py.File(filename, 'r') as h5file:
+        out = recursively_load_dict_contents_from_group(h5file, '/')
+        if ASLIST:
+            outl = [None for l in range(len(out.keys()))]
```

Ruff

Ambiguous variable name: `l`

(E741)